packages/ui-components/src/__tests__/StrategyPage.test.ts

```
@@ -15,51 +15,54 @@ vi.mock('@rainlanguage/orderbook/js_api', () => ({
 			}
 		});
+		await strategyPromise;
 
 		await waitFor(() => {
 			expect(screen.getByText('Test Strategy')).toBeInTheDocument();
 			expect(screen.getByText('Test Description')).toBeInTheDocument();
 		});
 	});
 
-	it('renders strategy details successfully from fetch', async () => {
+	it.only('renders strategy details successfully from fetch', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)